src/components/organisms/AdminVenueView/components/VenueRoomsEditor/VenueRoomsEditor.tsx

```diff
@@ -283,37 +281,36 @@ export const VenueRoomsEditor: React.FC<VenueRoomsEditorProps> = ({
             onClick={() => !selectedRoom && setSelectedRoom(room)}
           >
             <img
-              className={classNames("Container__room-image", {
-                "Container__room-image--disabled": !room.isEnabled,
-              })}
+              className={roomImageClass(room)}
               src={room.image_url}
               alt="room-logo"
               title={room.title}
             />
-            <div className="Container__room-title">{room.title}</div>
+            <div className="VenueRoomsEditor__room-title">{room.title}</div>
           </div>
         )
       ),
     [renderSelectedRoom, rooms, selectedRoom, setSelectedRoom]
   );
 
   return (
-    <div ref={drop} style={{ ...styles, ...containerStyle }}>
+    <div ref={drop} className="VenueRoomsEditor">
       <ReactResizeDetector
         handleWidth
         handleHeight
         onResize={(width, height) => setImageDims({ width, height })}
       />
-      {!backgroundImage ? (
-        <MapBackgroundPlaceholder />
+      {backgroundImage ? (
+        <div className="VenueRoomsEditor__background-image-wrapper">
+          <img
+            alt="draggable background"
+            className={backgroundImageClass}
```

Comment:
```suggestion
            className="VenueRoomsEditor__background-image"
```

Reply:
fixed in a4aa1c4f4ec90d75594111a0ed9e49045dfe7cd6